cli/cmd/import/import.go

```diff
@@ -0,0 +1,137 @@
+// Copyright AGNTCY Contributors (https://github.com/agntcy)
+// SPDX-License-Identifier: Apache-2.0
+
+package importcmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/agntcy/dir/cli/presenter"
+	ctxUtils "github.com/agntcy/dir/cli/util/context"
+	"github.com/agntcy/dir/importer/config"
+	"github.com/agntcy/dir/importer/mcp"
+	"github.com/agntcy/dir/importer/types"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cfg          config.Config
+	registryType string
+	factory      *types.Factory
+)
+
+var Command = &cobra.Command{
+	Use:   "import",
+	Short: "Import records from external registries",
+	Long: `Import records from external registries into DIR.
+
+Supported registries:
+  - mcp: Model Context Protocol registry v0.1
+
+The import command fetches records from the specified registry and pushes
+them to DIR.
+
+Examples:
+  # Import from MCP registry
+  dirctl import --type=mcp --url=https://registry.modelcontextprotocol.io
+
+  # Import with filters
+  # Available filters: https://registry.modelcontextprotocol.io/docs#/operations/list-servers-v0.1#Query-Parameters
+  dirctl import --type=mcp --url=https://registry.modelcontextprotocol.io --filter=updated_since=2025-08-07T13:15:04.280Z
+
+  # Preview without importing
+  dirctl import --type=mcp --url=https://registry.modelcontextprotocol.io --dry-run
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runImport(cmd)
+	},
+}
+
+func init() {
+	// Initialize factory and register importers once at startup
+	factory = types.NewFactory()
+	mcp.Register(factory)
+
+	// Add flags
+	Command.Flags().StringVar(&registryType, "type", "", "Registry type (mcp, a2a)")
```

Comment:
we can still register here the `cfg.RegistryType` without having a dedicated var for it, which we can validate below if the cfg.RegistryType is supported by config and available as importer